src/commands/modping.ts

```diff
@@ -0,0 +1,160 @@
+                }
+
+                const settings = await ModPingSettings.findOne({ where: { user_id: member.id } });
+                if (settings) {
+                    message += '\nAuto-assign will override this if you change statuses.';
+                }
+            } else {
+                message = 'Role not found.';
+            }
+        } else if (subcommandGroup === 'auto') {
+            const userId = interaction.user.id;
+            if (subcommand === 'assign') {
+                const online = interaction.options.getBoolean('online') ?? true;
+                const idle = interaction.options.getBoolean('idle') ?? true;
+                const dnd = interaction.options.getBoolean('do_not_disturb') ?? false;
+                const offline = interaction.options.getBoolean('offline') ?? false;
+                const statusList = [];
+                if (online) {
+                    statusList.push('Online');
+                }
+                if (idle) {
+                    statusList.push('Idle');
+                }
+                if (dnd) {
+                    statusList.push('Do Not Disturb');
+                }
+                if (offline) {
+                    statusList.push('Offline');
+                }
+
+                if (statusList.length === 0 || statusList.length === 4) {
+                    await interaction.reply({ content: `Sorry, this setup won't work. You need to have at least one status that isn't the same as the rest.`, ephemeral: true });
+                    return;
+                } else if (statusList.length === 1) {
+                    message = `<@&${roleId}> will be assigned when you are ${statusList[0]}.`;
+                } else {
+                    const lastStatus = statusList.pop();
+                    message = `<@&${roleId}> will be assigned when you are ${statusList.join(', ')} or ${lastStatus}.`;
+                }
+
+                const settings = {
+                    online,
+                    idle,
+                    dnd,
+                    offline
+                };
+
+                await ModPingSettings.upsert({
+                    user_id: userId,
+                    settings: JSON.stringify(settings)
+                });
+
+            } else if (subcommand === 'disable') {
+                message = 'Auto-assign has been disabled.';
+
+                await ModPingSettings.destroy({
+                    where: { user_id: userId }
+                });
+
+            } else if (subcommand === 'current') {
+                const settings = await ModPingSettings.findOne({
+                    where: { user_id: userId }
+                });
+
+                if (settings) {
+                    const { online, idle, dnd, offline } = JSON.parse(settings.settings);
+                    message = `Current settings:\n- ${online ? '✅' : '❌'} Online\n- ${idle ? '✅' : '❌'} Idle\n- ${dnd ? '✅' : '❌'} Do Not Disturb\n- ${offline ? '✅' : '❌'} Offline`;
```

Comment:
This would be easier to read.

```suggestion
                    message = 'Current settings:\n';
                    message += `- ${online ? '✅' : '❌'} Online\n`;
                    message += `- ${idle ? '✅' : '❌'} Idle\n`;
                    message += `- ${dnd ? '✅' : '❌'} Do Not Disturb\n`;
                    message += `- ${offline ? '✅' : '❌'} Offline`;
```

Reply:
i did change it but not in this way because i changed the structure of the command